lib/liftoff/cli.rb

```diff
@@ -18,7 +18,7 @@ def parse_command_line_options
 
     def global_options
       OptionParser.new do |opts|
-        opts.banner = 'usage: liftoff [-v | --version] [-h | --help] [config options]'
+        opts.banner = 'usage: liftoff [-v | --version] [-h | --help] [config options] [path]'
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.